bundle/file.go

```diff
@@ -343,6 +362,11 @@ func (t *tarballLoader) NextFile() (*Descriptor, error) {
 					}
 				}
 
+				if t.maxSizeLimitBytes > 0 && header.Size > t.maxSizeLimitBytes {
```

Comment:
The description for the `header.Size` is `Logical file size in bytes` and `Header.Size determines how many bytes can be read for the next file`. It seems this would reflect the actual file size, correct?